test/config/300-serverlessservice.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 The Knative Authors
```

Comment:
2021.
Is there anything we need to keep in sync?

Reply:
Already sent the respective PRs to clean up the copied CRDs.